web/cobrands/sass/_base.scss

```diff
@@ -1957,6 +1957,85 @@ html.js #map .noscript {
   @include svg-background-image("/cobrands/fixmystreet/images/map-tools");
 }
 
+// Map instructions for keyboard
+.keyboard-intructions-wrapper {
+  display:flex;
+  position: absolute;
+  padding: 10px 25px;
+  padding-top: 14px;
+  background-color: white;
+  border-radius: 8px;
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+  bottom: 35px;
+  left: 50%;
+  transform: translateX(-50%);
+
+  span {
+    display: inline-block;
+    margin-left: 20px;
+    font-weight: 600;
+  }
+
+  &.basic {
+    opacity: 0;
+  }
+
+  @media only screen and (max-width:48em) {
```

Comment:
Oooh, just noticed what you’re doing here. Rather than including stuff in `_base.scss` but hiding it on narrow screens with a media query, we tend to just move the stuff into `_layout.scss` instead, which only runs on wider screens to begin with.

I guess we should think about whether this keyboard prompt really _should_ be hidden on narrow screens, though. I assume the "mobile" map view will still be keyboard navigable. Thinking about it logically, the only time we’d really want to _hide_ the keyboard prompt is when _there is no keyboard_. I wonder whether we can detect that via JavaScript… 🤔